src/less/textbox/ds6/textbox-mixin.less

```diff
@@ -0,0 +1,70 @@
+.textbox-mixin() {
+    @import "../../../../dist/textbox/ds6/textbox.css";
+}
+
+.placeholder() {
```

Comment:
The mixins should not be in this file. See my other comment about creating a `textbox-mixins.less` (plural) file.